<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Надёжная программа</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Solver123Button">
      <property name="text">
       <string>Решение задач на количественный отказ(1,2,3)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Solver45Button">
      <property name="text">
       <string>Решение задач на среднее время(4,5)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Solver6Button">
      <property name="text">
       <string>Решение задачи с таблицей(6)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
